pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -655,6 +689,13 @@ async def _get_event_iterator(self) -> AsyncIterator[ModelResponseStreamEvent]:
                 pass
 
             elif isinstance(event, BetaRawContentBlockStopEvent | BetaRawMessageStopEvent):  # pragma: no branch
+                # Capture mapped finish reason on message stop
+                try:
+                    raw_reason = getattr(getattr(event, 'message', None), 'stop_reason', None)
```

Comment:
We should not need the `getattr` here, the types ensure these fields are available